expense-notifications-functions/lib/index.js

```
@@ -0,0 +1,404 @@
+const firestore_1 = require("firebase-admin/firestore");
+const firestore_2 = require("firebase-functions/v2/firestore");
+const admin = require("firebase-admin");
+const logger = require("firebase-functions/logger");
+admin.initializeApp();
+const db = (0, firestore_1.getFirestore)();
+const notificationTitle = `Splito`;
+// Cloud Function to handle new expense creation and notify users
+exports.onExpenseCreated = (0, firestore_2.onDocumentCreated)({ document: 'groups/{groupId}/expenses/{expenseId}' }, async (event) => {
+    var _a;
+    try {
+        const expenseData = (_a = event.data) === null || _a === void 0 ? void 0 : _a.data();
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)